targets/TARGET_STM/PinNamesTypes.h

```diff
@@ -82,6 +82,10 @@ extern "C" {
 #define STM_PIN_AN_CTRL_SHIFT 20
 #define STM_PIN_ANALOG_CONTROL_BIT (STM_PIN_AN_CTRL_MASK << STM_PIN_AN_CTRL_SHIFT)
 
+#define STM_PIN_AN_CHAN_BANK_B_MASK 0x01
+#define STM_PIN_AN_CHAN_BANK_B_SHIFT 22
```

Comment:
Please update the beginning of this file (line 50)

PS: why bit 22 is used and not 21 ? :-)

Reply:
Corrected and Updated line 50.